ledger/sandbox/src/main/resources/db/migration/postgres/V12__package_entries.sql

```diff
@@ -0,0 +1,22 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+---------------------------------------------------------------------------------------------------
+-- V12: package_entries for package upload
+--
+-- This schema version adds a table for tracking DAML-LF package submissions
+-- It includes id to track the package submission and status
+---------------------------------------------------------------------------------------------------
+
+
+ALTER TABLE packages
+    -- SubmissionId for package to be uploaded
+    ADD submission_id varchar not null,
```

Comment:
I don't think this approach works. You need a separate table for the package uploads as one package upload can upload multiple packages. I would add a table called "package_upload_entries" that records what happened to a submission.

Reply:
@dajmaki The thought was not wanting to duplicate the binary payload in yet another table as we will now always store whatever we have received.

But yes if we want to treat status of the multiple packages as one atomic entry the separate table would be cleaner

And would we still keep a duplicate in the packages table in its existing form to be used by listPackages and related api calls?

Reply:
There's no need to duplicate they payload. One table contains the actual packages and the other contains the submission results. You don't need the payload in the submission results table, you can just list the package ids that were actually uploaded by the submission.

Reply:
now created a separate package upload entries table (and party allocation entries table) that will store both accept and reject